src/Fields/Fields.jl

```diff
@@ -332,4 +332,20 @@ function level(field::FaceExtrudedFiniteDifferenceField, v::PlusHalf)
     Field(data, hspace)
 end
 
+"""
+    set!(field::Field, f::Field)
```

Comment:
Thanks Charlie. Does it matter in this signature which one is the input/output Field? If so, do you mind slightly changing the docstring so that this is clearly documented?